roles/schema/tasks/schemas_override.yml

```diff
@@ -0,0 +1,93 @@
+---
+- set_fact:
+    sr_user: "{{ schema_registry_ldap_user if schema_registry_ldap_user is defined else sasl_plain_users.schema_registry.principal  }}"
+
+- set_fact:
+    sr_user_password: "{{ schema_registry_ldap_password if schema_registry_ldap_password is defined else sasl_plain_users.schema_registry.password  }}"    
+
+- set_fact:
+    serverlist: "{{ groups[target_group]  }}" 
+  vars:
+    target_group: schema_registry
+
+- set_fact:
+    target_host: "{{ groups[target_group][0]  }}"     
+  vars:
+    target_group: schema_registry    
+
+- set_fact:
+    protocol: "{{ 'https://' if tls_enabled is true else 'http://' }}"
+
+- name: Build SR URL
+  set_fact:
+       sr_server_url: "{{ protocol + groups[target_group][0] + ':' + sr_internal_port }}"
+  vars:
+    target_group: schema_registry  
+
+- name: Fetch Certificate Key Files from Remote Node to Controller for mTLS scenarios
+  run_once: yes
+  fetch: src={{ ssl_file_path }}/{{host_key_name}} dest=/tmp/ flat=yes
```

Comment:
Please make the /tmp path configurable with /tmp being default.

Reply:
This location is now made configurable via the variable local_ssl_file_path.